src/Poseidon/Package.hs

```diff
@@ -200,15 +200,20 @@ data PoseidonPackage = PoseidonPackage
     deriving (Show, Eq, Generic)
 
 data PackageReadOptions = PackageReadOptions
-    { _readOptVerbose          :: Bool -- whether to print verbose output
-    -- whether to stop on duplicated individuals
-    , _readOptStopOnDuplicates :: Bool -- whether to stop on duplicated individuals
-    -- whether to ignore all checksums
-    , _readOptIgnoreChecksums  :: Bool -- whether to ignore all checksums
-    -- whether to ignore missing genotype files, useful for developer use cases
-    , _readOptIgnoreGeno       :: Bool -- whether to ignore missing genotype files, useful for developer use cases
-    -- whether to check the first 100 SNPs of the genotypes
-    , _readOptGenoCheck        :: Bool -- whether to check the first 100 SNPs of the genotypes
+    { _readOptVerbose          :: Bool
+    -- ^ whether to print verbose output
+    , _readOptStopOnDuplicates :: Bool
+    -- ^ whether to stop on duplicated individuals
+    , _readOptIgnoreChecksums  :: Bool
+    -- ^ whether to ignore all checksums
+    , _readOptIgnoreGeno       :: Bool
+    -- ^ whether to ignore missing genotype files, useful for developer use cases
+    , _readOptGenoCheck        :: Bool
+    -- ^ whether to check the first 100 SNPs of the genotypes
+    , _readOptIgnorePacVersion :: Bool
```

Comment:
OK, so this is new... but I don't get why you ignore that in the update subcommand?

Reply:
As explained above: To update from 2.4.0 to 2.5.0 `update` has to be allowed to read 2.4.0.